src/builder/Actions.sol

```diff
@@ -100,6 +111,7 @@ library Actions {
         uint256 price;
         address recipient;
         address token;
+        string assetSymbol;
```

Comment:
nit: put in alphabetical order relative to the other fields (below `amount`)

Reply:
Done